packages/grafana-data/src/types/datasource.ts

```diff
@@ -535,19 +542,19 @@ export interface DataSourceSelectItem {
 /**
  * Options passed to the datasource.annotationQuery method. See docs/plugins/developing/datasource.md
  */
-export interface AnnotationQueryRequest<MoreOptions = {}> {
+export interface AnnotationQueryRequest<TAnno = {}> {
```

Comment:
would be nice if TAnno *extends* BaseAnnotationQuery... but that will break many existing types that use TQuery for the annotation type